modules/storages/app/components/storages/admin/side_panel/health_notifications_component.rb

```diff
@@ -55,6 +57,31 @@ def notification_status
             description: I18n.t("storages.health_email_notifications.description_unsubscribed") }
         end
       end
+
+      def health_status_indicator
+        case @storage.health_status
+        when "healthy"
+          { scheme: :success, label: I18n.t("storages.health.label_healthy") }
+        when "unhealthy"
+          { scheme: :danger, label: I18n.t("storages.health.label_error") }
+        else
+          { scheme: :attention, label: I18n.t("storages.health.label_pending") }
+        end
+      end
+
+      # This method returns the health identifier, description and the time since when the error occurs in a
+      # formatted manner. e.g. "Not found: Outbound request destination not found since 12/07/2023 03:45 PM"
+      def formatted_health_reason
+        identifier = @storage.health_reason_identifier.tr("_", " ").strip
+        description = @storage.health_reason_description
+
+        if description.present?
+          identifier.capitalize!
+          identifier << ": #{description}"
+        end
+
+        "#{identifier} #{I18n.t('storages.health.since', datetime: helpers.format_time(@storage.health_changed_at))}"
```

Comment:
Two notes:

:red_circle: We should definitely make the `identifier` part of the translation, e.g.

```suggestion
        I18n.t("storages.health.since", identifier:, datetime: helpers.format_time(@storage.health_changed_at))
```

Otherwise we enforce the "english ordering" onto all other languages, regardless of whether it makes sense there. E.g. imagine in a language it must be written as "Since DATE ERROR_MESSAGE", this is only possible by moving the error message into the translation.

:yellow_circle: To avoid confusion, I'd actually change the sentence.

```diff
- Not found: Outbound request destination not found since 12/07/2023 03:45 PM
+ Not found: Outbound request destination not found. Storage unhealthy since 12/07/2023 03:45 PM
```

The reason for that is, that we only know that the storage is unhealthy since a given date, not that it's the same error occuring all the time.